Dockerfile

```diff
@@ -57,6 +58,7 @@ ENV USER ansible
 
 # Add contents of the DebOps monorepo to the container
 COPY . .local/share/debops/debops
+RUN sudo chown -R ansible:ansible /home/ansible/.local
```

Comment:
@reixd Don't use `sudo` here, Dockerfile already work with `root` permissions.

Reply:
I was using this image as the controller and got some issues with the permission under the given path. So I thought to fix it.
Yes its true, docker works as root, but after setting `USER ansible` not anymore, therefore the need to use `sudo`.

Reply:
OK, I checked in a Docker container and the `/home/ansible/.local/` directory and subdirectories are owned by `root` account, not `ansible`. I thought that by placing the `COPY` command after the `USER` command the files would be owned by `ansible:ansible`, I gues that changed at some point? Anyway, the solution would be to move the `COPY` command before the `USER` command, `RUN` a `chown` command afterwards without `sudo`, and then switch to `USER ansible`.

Reply:
> Anyway, the solution would be to move the `COPY` command before the `USER` command, `RUN` a `chown` command afterwards without `sudo`, and then switch to `USER ansible`.

That would create many more image layers, making the docker image bigger.

RTFM helps, I just found out what's needed:  `COPY --chown=patrick hostPath containerPath`
See https://docs.docker.com/engine/reference/builder/#copy